packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -168,6 +168,22 @@ enum SKProductDiscountPaymentMode {
   unspecified,
 }
 
+/// Dart wrapper around StoreKit's [SKProductDiscountType](https://developer.apple.com/documentation/storekit/skproductdiscounttype?language=objc)
+///
+/// This is used as a property in the [SKProductDiscountWrapper].
+/// The values of the enum options are matching the [SKProductDiscountType]'s
+/// values. Should there be an update or addition
+/// in the [SKProductDiscountType], this need to be updated to match.
```

Comment:
nit: this could be wrapped a little bit more concise:

```suggestion
/// Dart wrapper around StoreKit's [SKProductDiscountType](https://developer.apple.com/documentation/storekit/skproductdiscounttype?language=objc)
///
/// This is used as a property in the [SKProductDiscountWrapper].
/// The values of the enum options are matching the [SKProductDiscountType]'s
/// values. Should there be an update or addition in the 
/// [SKProductDiscountType], this need to be updated to match.
```

Reply:
Done